### PR TITLE
Support plain config instead of zipping

### PR DESCRIPTION
**Pull request by hiboyang**
## What is the Purpose of the Change

The gateway has config file, which was previously required to be zipped. Add plain config to support unzipped config, making it easy to use.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Brief Change Log

The gateway has config file, which was previously required to be zipped. Add plain config to support unzipped config, making it easy to use.

## Verifying This Change

Manually tested

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

# Final Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: helm/batch-processing-gateway/values.yaml [hunk 1]
```diff
@@ -22,7 +22,128 @@ ingress:
...
+    spark.kubernetes.submission.connectionTimeout: 30000
+    spark.kubernetes.submission.requestTimeout: 30000
+    spark.kubernetes.driver.connectionTimeout: 30000
+    spark.kubernetes.driver.requestTimeout: 30000
+    spark.sql.debug.maxToStringFields: 75
+  sparkClusters:
+    - weight: 100
+      id: c0101
+      eksCluster: cluster-1
+      masterUrl: https://34.134.183.114
```
Comment (tongtianqi777) [inline helm/batch-processing-gateway/values.yaml:38]: is this a dummy value? can you make it an empty string or make it more obvious that it's a dummy value?

File: helm/batch-processing-gateway/values.yaml [hunk 2]
```diff
@@ -22,7 +22,128 @@ ingress:
...
+      executorNodeLabelKey: "node_group_category"
+      executorNodeLabelValues:
+        - "spark_executor"
+    - name: vip
+      secure: true
+  queueTokenSOPS:
+    secrets:
+      - secret$$$1
+  dbStorageSOPS:
+    connectionString: jdbc:mysql://localhost:3306/skate?useUnicode=yes&characterEncoding=UTF-8&useLegacyDatetimeCode=false&connectTimeout=10000&socketTimeout=30000
```
Comment (tongtianqi777) [inline helm/batch-processing-gateway/values.yaml:134]: please rename the `skate` here to something else

File: helm/batch-processing-gateway/values.yaml [hunk 3]
```diff
@@ -22,7 +22,128 @@ ingress:
...
+    user: root
+    password: Pass1word
+    dbName: skate
```
Comment (tongtianqi777) [inline helm/batch-processing-gateway/values.yaml:137]: also rename the skate here please

**Review by tongtianqi777 — CHANGES_REQUESTED** [review]

File: helm/batch-processing-gateway/values.yaml (hunk 1 above)
Comment (hiboyang) [reply helm/batch-processing-gateway/values.yaml:38]: yes, should be a dummy value

File: helm/batch-processing-gateway/values.yaml (hunk 2 above)
Comment (hiboyang) [reply helm/batch-processing-gateway/values.yaml:134]: yes, good catch

File: helm/batch-processing-gateway/values.yaml (hunk 3 above)
Comment (hiboyang) [reply helm/batch-processing-gateway/values.yaml:137]: yes

**Review by tongtianqi777 — APPROVED** [review]

**Pull request merged**